docs/notebooks/bsync_examples/Primary-School-Levels1-2.md

```diff
@@ -0,0 +1,8205 @@
+    gym_sec,
+    kitchen_sec,
+    lib_sec,
+    mech_sec,
+    office_sec,
+    wb_sec
+]
+for section in sections:
+    section += bsync.Roofs(
+        bsync.Roof(
+            bsync.RoofID(
+                bsync.RoofArea(float(section.FloorAreas.FloorArea[1].FloorAreaValue._text)),
+                bsync.RoofCondition("Good"),
+                IDref = 'Roof'
+            )
+        )
+    )
+```
+
+##### 6.2.3 Walls and Windows
+
+For characterizing the envelope, we will define wall, window, foundation and ceiling systems, then define sides to each section, and finally assign these systems to each section.
+
+###### 6.2.3.1 Walls
+
+We define the wall system:
+
+
+```python
+wall_sys = bsync.WallSystems()
+wall = bsync.WallSystem(
+    bsync.ExteriorWallConstruction("Steel frame"),
+    bsync.ExteriorWallFinish("Other"),
+    bsync.WallRValue(8.063),
+    bsync.WallExteriorSolarAbsorptance(0.7),
+    bsync.WallExteriorThermalAbsorptance(0.9),
+    bsync.ExteriorRoughness("Smooth"),
+    ID="Wall"
+)
+wall_sys += wall
+systems += wall_sys
+```
+
+###### 6.2.3.2 Windows and Doors
+
+We define the windows and doors system:
+
+
+```python
+window_sys = bsync.FenestrationSystems()
+doors = bsync.FenestrationSystem(
+    bsync.FenestrationType(
+        bsync.Door(
+            bsync.ExteriorDoorType("Uninsulated metal"),
+            bsync.Vestibule(False),
+            bsync.DoorOperation("NonSwinging"),
+            bsync.DoorGlazedAreaFraction(0.2)
+        )
+    ),
+    bsync.FenestrationFrameMaterial("Clad"),
+    bsync.FenestrationOperation(True),
+    bsync.GlassType("Tinted plus low e"),
+    bsync.FenestrationGasFill("Air"),
+    bsync.FenestrationGlassLayers("Double pane"),
+    bsync.FenestrationUFactor(0.571),
+    ID="Doors"
+)
+
+windows = bsync.FenestrationSystem(
+    bsync.FenestrationType(
+        bsync.Window(
+            bsync.WindowLayout("Continuous"),
+            bsync.WindowSillHeight(3.6),
+            bsync.ExteriorShadingType("None"),
+            bsync.InteriorShadingType("None")
+        )
+    ),
+    bsync.FenestrationFrameMaterial("Clad"),
+    bsync.FenestrationOperation(True),
+    bsync.GlassType("Tinted plus low e"),
+    bsync.FenestrationGasFill("Air"),
+    bsync.FenestrationGlassLayers("Double pane"),
+    bsync.FenestrationUFactor(0.571),
+    ID="Windows"
+)
+
+window_sys += doors
+window_sys += windows
+systems += window_sys
+```
+
+###### 6.2.3.3 Foundations
+
+We define the building foundations:
+
+
+```python
+foundation_sys = bsync.FoundationSystems()
+foundation = bsync.FoundationSystem(
+    bsync.GroundCouplings(
+        bsync.GroundCoupling(
+            bsync.SlabOnGrade(
+                bsync.SlabInsulationOrientation("None"),
+                bsync.SlabHeating("Unheated")
+            )
+        )
+    ),
+    bsync.FloorCovering("Carpet"),
+    bsync.FloorConstructionType("Built up"),
+    ID="Foundation"
+)
+foundation_sys += foundation
+systems += foundation_sys
+```
+
```

Comment:
Do we need to add in overall enclosure tightness as well?

Reply:
I have that in the AirInfiltrationSystems near the end. Do you mean that?

Reply:
Ah ok - I did not realize it was the same thing,  Thanks for clarifying.